XSLT-to-SVG/temi_xslt_svg/blobs_to_svg.xsl

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns:math="http://www.w3.org/2005/xpath-functions/math"
+    exclude-result-prefixes="xs math"
+    version="3.0">
+    
```

Comment:
Now I'd love to see the XSLT pull the data and make the shapes for the SVG...*and* provide the legend...